compiler/scenario-service/client/src/DA/Daml/LF/ScenarioServiceClient/LowLevel.hs

```diff
@@ -205,7 +206,13 @@ withScenarioService opts@Options{..} f = do
   unless serverJarExists $
       throwIO (ScenarioServiceException (optServerJar <> " does not exist."))
   validateJava
-  cp <- javaProc (optJvmOptions <> ["-jar" , optServerJar] <> maybeToList (show <$> optGrpcMaxMessageSize))
+  cp <- javaProc $ concat
+    [ optJvmOptions
+    , ["-jar" , optServerJar]
+    , ["--max-inbound-message-size:" <> show size | Just size <- [optGrpcMaxMessageSize]]
```

Comment:
```suggestion
    , ["--max-inbound-message-size=" <> show size | Just size <- [optGrpcMaxMessageSize]]
```

Reply:
ah, that was actually what I had before, but the examples at https://github.com/scopt/scopt all use either a colon or a space, never an equals sign. I should test it directly on the executable, though

Reply:
both work on the executable (no idea why they don't show it in the examples?) so I'll switch to `=`s